src/System.Runtime.InteropServices.RuntimeInformation/src/System.Runtime.InteropServices.RuntimeInformation.builds

```diff
@@ -15,13 +15,22 @@
       <OSGroup>Windows_NT</OSGroup>
     </Project>
     <Project Include="System.Runtime.InteropServices.RuntimeInformation.csproj">
-      <TargetGroup>net46</TargetGroup>
+      <OSGroup>Windows_NT</OSGroup>
+      <TargetGroup>net45</TargetGroup>
+      <PackageTargetFramework>net45</PackageTargetFramework>
+      <AdditionalProperties>PackageTargetPath=runtimes/win7/lib/net45</AdditionalProperties>
```

Comment:
Don't set this.  Especially not win7.  Use PackageTargetRuntime in the csproj as 'win'.  Similar to below.